### DISCUSSION TITLE
Question About System BIOS

### DISCUSSION THREAD

**Discussion by EricWilliamYT** [EmulatorJS/EmulatorJS#720] [Q&A] [answered]
So, For Systems That Support The Use Of Bios. Can I Have Just The Bios And Nothing Else, Or Do I HAVE To Include A Game.

**Comment by ethanaobrien** [comment, marked as answer]
Yes, you need to include a game. Without a game, the emulator has nothing to run

**Reply by EricWilliamYT** [reply]
ok, thanks for letting me know.
